list_of_agents:
 - 'master'
 - 'robot1'
 - 'robot2'
